src/OrbitGl/CaptureViewElement.cpp

```diff
@@ -11,9 +11,7 @@ namespace orbit_gl {
 
 CaptureViewElement::CaptureViewElement(CaptureViewElement* parent, const Viewport* viewport,
                                        const TimeGraphLayout* layout)
-    : viewport_(viewport), layout_(layout), parent_(parent) {
-  ORBIT_CHECK(layout != nullptr);
```

Comment:
Yes, but no one in CaptureViewElement.h/cpp. So, It can work with nullptr there. It's needed for the tests. Otherwise I have to provide a fake layout in every FakeSlider construction.

Reply:
> Otherwise I have to provide a fake layout in every FakeSlider construction.

Which I think you should do. We have the same thing with the `nullptr` we're passing in for OrbitApp, and I think this is really a workaround because it would be hard to construct OrbitApp. I'd like to get rid of the `nullptr` constructor arguments unless they are part of the design (e.g. `parent` can explicitly be a `nullptr` to indicate we're at the root of the tree).

Reply:
Reverted :)